src/main/java/com/kodilla/ecommercee/exception/GroupException.java

```diff
@@ -0,0 +1,8 @@
+package com.kodilla.ecommercee.exception;
+
+
+public class GroupException extends Exception {
```

Comment:
GroupException should be handled in GlobalError class and it's name should be different than just GroupException so anyone can know what is wrong with the application.

Reply:
okk, changed